File: name-score/src/main/java/com/shrader/namescore/Application.java
package com.shrader.namescore;

import lombok.extern.log4j.Log4j2;
import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication
@Log4j2
public class Application implements CommandLineRunner {

	public static void main(String[] args) {
		SpringApplication.run(Application.class, args);
	}

	@Override
	public void run(String... args) {
		log.info("Welcome:");
	}
}

File: name-score/name-score-api/src/main/kotlin/com/shrader/namescore/api/Application.kt
package com.shrader.namescore.api

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.ApplicationArguments
import org.springframework.boot.ApplicationRunner
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.ApplicationContext
import java.util.*


@SpringBootApplication
open class Application : ApplicationRunner {

    @Autowired
    private val applicationContext: ApplicationContext? = null

    @Override
    @Throws(Exception::class)
    override fun run(args: ApplicationArguments) {
        val beans = applicationContext!!.beanDefinitionNames
        Arrays.sort(beans)
        for (bean in beans) {
            println(bean)
        }
    }

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            runApplication<Application>(*args)
        }
    }
}
